src/components/FeatureInformation/FeatureInformation.js

```diff
@@ -82,6 +82,9 @@ const FeatureInformation = ({ clickedFeatureInfo }) => {
             className="wkp-close-bt"
             onClick={() => {
               dispatch(setClickedFeatureInfo());
+              if (PopupComponent.actionOnClose) {
+                PopupComponent.actionOnClose();
```

Comment:
DidUnmount  is not called on DeparturePopup when we close the popup with the close button ?

Reply:
'componentWillUnmount' is called when clicking on the close button, but as well on clicking outside of the popup.
This solution allows us to return to the netzkarte Popup only on click on close button.